### DISCUSSION TITLE
Unable to run postal bootstrap and postal initialize

### DISCUSSION THREAD

**Discussion by anthonyolazabal** [postalserver/postal#1881] [Help with using Postal]
Hello, 

I'm preparing a brand new installation of Postal with the latest sources, but I'm facing an issue on the first steps of the documentation. 
I'm unable to run the postal bootstrap command and also postal initialize. I'm having the following issue :
postal initialize
Initializing database
**./postal: line 6: bundle: command not found**

I'm running Ubuntu 20.04 with the latest updates.

Thanks in advance for your help

**Comment by willpower232** [comment]
how have you installed postal? which instructions did you follow?

**Reply by anthonyolazabal** [reply]
Ok I figure it out, I can't use directly the postal from the main postal repo. I need to use the bash script from the install repository.
